data/ka'het/ka'het missions.txt

```diff
@@ -86,12 +117,49 @@ mission "First Contact: Ka'het: Remnant 1B"
 			choice
 				`	"Wait, why did you call them 'shells' and not ships?"`
 			label shells
-			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. This could have happened in the Graveyard, since it's the only place where we know them to live in, but we have such limited data about that section of space that we can't say for sure." As she finishes talking, one of the people Taely was talking to earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
+			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. We call them the Builders. After we lost the first scout ships in the Graveyard, the prefects decided that we should continue to scout the region but take necessary precautions. Thanks to those protocols, some of which are still applied today, we were able to extensively explore most of the Graveyard and decipher a good amount of the written language we found. There is still much we have yet to discover." You discuss the Graveyard with Taely for a while longer until one of the people she was talking to earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
 				decline
 	on decline
+		event "graveyard insights"
 		log `Asked Taely about the Graveyard and the Ka'het and learned of the Remnant's previous contacts between them.`
 		log "Factions" "Ka'het" `The Ka'het are a species which live in the Graveyard, an area south of the Ember Waste. They are large lifeforms which can travel in space using artificial exoskeletons and will attack anyone on sight, even other Ka'het.`
 
+mission "graveyard insights compatibility"
+	landing
+	invisible
+	to offer
+		has "First Contact: Ka'het: Remnant 1B: offered"
+		not "event: Graveyard insights"
+	on offer
+		event "graveyard insights"
+		fail
+
+event "graveyard insights"
+	planet Alix
+		description `	 Here the Ka'het attacks were limited to the inhabited settlements; had they also destroyed the mining bases and the unmanned vessels, station Parii'het would have run out of supplies thousands of years ago.`
```

Comment:
These should be `add description`, as otherwise these are overwriting the existing descriptions with what is in this event.